README.md

```diff
@@ -15,35 +15,16 @@
 [![Dependencies status](https://badgen.net/david/dep/cerner/terra-dev-site)](https://david-dm.org/cerner/terra-dev-site)
 [![devDependencies status](https://badgen.net/david/dev/cerner/terra-dev-site)](https://david-dm.org/cerner/terra-dev-site?type=dev)
 
-Dynamically builds a site based on site configuration, navigation configuration and page configuration.
+A webpack plugin to dynamically build a documentation and testing site using terra-application as the foundation. See [terra-ui](https://engineering.cerner.com/terra-ui/dev_tools/terra-dev-site/terra-dev-site/about) for documentation.
 
-The provided webpack plugin includes a call to the generateAppConfig script. The generateAppConfig script builds out static config to the ./dev-site-config/build folder. generateAppConfig also discovers pages based on it's configuration. After the static config has been built webpack continues to run, pulling in the static config, and producing the webpack bundle.
+## Installing the package
 
-Provides the following default configuration:
+```bash
+npm install --save-dev @cerner/terra-dev-site
 
-* `config/site/site.config.js`
-* `config/site/navigation.config.js`
-* `config/webpack/webpack.config.js`
-
-* [Getting Started](#getting-started)
-* [Versioning](#versioning)
-* [Contributing](#contributing)
-* [LICENSE](#license)
-
-## Getting Started
-
-* Install with [npm](https://www.npmjs.com): `npm install --save-dev @cerner/terra-dev-site`
-
-## Peer Dependencies
-
-This component requires the following peer dependencies be installed in your app for the component to properly function.
-
-| Peer Dependency | Version |
-|-|-|
-| react | ^16.8.5 |
-| react-dom | ^16.8.5 |
-| terra-toolkit | ^5.2.0 |
-| webpack | ^4.28.1 |
+// Install Peer Dependencies
+npm install --save-dev @cerner/terra-application react-dom react webpack
```

Comment:
This will install webpack 5 and react 17.

Not sure if we can do:

npm install --save-dev @cerner/terra-application react-dom@^16.8.5 react@^16.8.5 webpack@^4.28.1

Reply:
lol